graphql/resolvers.go

```diff
@@ -50,6 +50,27 @@ func (r *mutationResolver) AddFavoriteProject(ctx context.Context, identifier st
 	}, nil
 }
 
+func (r *mutationResolver) RemoveFavoriteProject(ctx context.Context, identifier string) (*restModel.UIProjectFields, error) {
+	p, err := model.FindOneProjectRef(identifier)
+	if err != nil || p == nil {
+		return nil, errors.Errorf("could not find project '%s'", identifier)
+	}
```

Comment:
I'd return separate errors here. (one should be a 500 and one should be a 404

Reply:
Errors work a bit differently in graphql than in HTTP. I will add codes to the errors but they will be in graphql speak, not in HTTP. Apollo server does a good job creating gql error standards in their [docs](https://www.apollographql.com/docs/apollo-server/data/errors/). I will add better error codes like `INTERNAL_SERVER_ERROR` and `RESOURCE_NOT_FOUND` alongside the error message.

Reply:
cool, there's just semantic differences between both of these errors that this implementation doesn't capture in the current form.

Reply:
updated errors to have codes and created a ticket to implement error system for all resolver errors: https://jira.mongodb.org/browse/EVG-7342